test/game/game_setup_test.exs

```diff
@@ -6,14 +6,41 @@ defmodule GameSetupTest do
   alias Game.Players
   alias Game.PlayerType.HumanPlayer
   alias Game.PlayerType.EasyComputerPlayer
+  alias Communication.CommandLine
+
+  @mock_command_line %CommandLine{communicator: CommunicatorMock}
 
   describe "setup_players/1" do
-    test "returns players containing both player's with their types that were determined via user input and symbols" do
+    test "returns players containing a Human player 1 and Human player 2 with their assosciated symbols" do
+      user_inputs = ["1", "1"]
```

Comment:
I wonder if there's a way we can make the input here more explicit. Because user input determines how this function behave and affects the test outcome, it'll be easier for anyone coming into the codebase to understand what these inputs mean without having to rely on memory or past experience. For example:
```
...
user_inputs = [select_to_play_with_computer, select_to_play_with_an_easy_computer]
...
```

Reply:
+1 @mrfishball

Reply:
For the record, Jon had a question about the comment I made above and I didn't get to address the question in time so I asked Jon to go ahead with the merge and made the suggested change in future PRs.